cc3d/CompuCellSetup/simulation_setup.py

```diff
@@ -580,6 +580,10 @@ def main_loop_player(sim, simthread=None, steppable_registry=None):
 
     cur_step = beginning_step
 
+    # initial drawing - we use mcs = -2 as a sentinel for mcs right after start function of steppables is completed
+    simthread.loopWork(-2)
```

Comment:
Any way we could formalize this?

Reply:
Specifically, using a defined variable instead of hard-coding a value for the flag.